client/app/main/dashboard/dashboard.controller.js

```diff
@@ -55,6 +62,7 @@ function DashboardCtrl($scope, $timeout, AbsenceRecord, GridDefaults, Student) {
     text: 'Withdraw selected',
     action: function() {
       updateSelected('withdrawn', true).then(function() {
+        $scope.$broadcast('withdrawn-updated');
```

Comment:
$emit dispatches an event upwards, $broadcast dispatches an event downwards. Since we are only listening for the event in the current scope we can use either.